package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Remove this. We use yarn and it creates a yarn.lock file 👍

Reply:
ok